modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -2448,6 +2448,13 @@ AUI.add(
 							layoutPath.groupId,
 							layoutPath.privateLayout
 						);
+						if (index < layoutsPathLength - 1) {
```

Comment:
@javalosjr96, can you please add a line break before this `if`?

Reply:
The source format fails if a line break is added.

Reply:
This is from the source formatter, it fails otherwise.